src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
Remove this file as we won't need it for this project. Read when serviceWorker is used  for your own information